src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -14,30 +14,38 @@
   private Boolean forceHttps;
   private Boolean forceLanguageInPath;
   protected static final String HTTP_PREFIX = "http://";
+  private String language;
 
   public UrlBuilder() {
     this.env = Configuration.getEnv();
     this.envType = Configuration.getEnvType(this.env);
     this.forceHttps = Configuration.getForceHttps();
     this.forceLanguageInPath = Configuration.getForceLanguageInPath();
+    this.language = Configuration.getWikiLanguage();
+  }
+
+  public UrlBuilder(String language) {
```

Comment:
No.
If find it messy that there is a constructor that takes a string as a parameter. and below there is a very similar one that takes a string and a bool, but the string is an `env` instead of the `language`. Do you need so many constructors and params?
You could do a private constructor that takes all the params and add a few static builder methods for specific usages. But I think it will be even cleaner if you move the wiki name to UrlBuilder constructor params.
This way you can have two static builder methods like `CreateBuilder` that takes all the values from the config and `CreateBuilderForWiki` that takes wiki name as a param and enforces `en` as the language.